viewing_party/party.py

```diff
@@ -0,0 +1,192 @@
+# ——— WAVE 01 ———
+def create_movie(title, genre, rating):
```

Comment:
👍 This works, it's doing a bit more than the tests require, but it does satisfy the requirements.